adminPanel/adminPanel.php

```diff
@@ -0,0 +1,36 @@
+<?php
+$db = new PDO("mysql:host=192.168.20.20;dbname=PortfolioProjects", 'root', '');
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$sql="SELECT `title` FROM `projects`;";
+$query = $db->query($sql);
+$titles = $query->fetchAll();
+?>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio Liliya Voevodina</title>
+        <link href="../css/normalize.css" rel="stylesheet" type="text/css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="../css/styleAdminPanel.css" rel="stylesheet" type="text/css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+ <body>
+    <h1>Projects management panel</h1>
+      <?php
+        echo '<td><a href="#">Add new projet</a></td>';
+        echo '<br><br>';
```

Comment:
avoid using <br> tags for styling. Preference adding a class and margin-bottom in css

Reply:
got it